plugins/pastefromword/filter/default.js

```diff
@@ -556,6 +563,21 @@
 			}
 		},
 
+		// Maps common attributes to styles.
+		mapCommonStyles: function( element ) {
+			return Style.mapStyles( element, {
+				vAlign: function( value ) {
+					Style.setStyle( element, 'vertical-align', value );
+				},
+				width: function( value ) {
```

Comment:
What if `width` or `height` were set in percentages, not in pixels?

Reply:
AFAIK World/Excel uses pixels for width/height data attributes, at least for images and tables. I'm not eager for premature implementation to keep the code simple. Do you have some verification that these attributes may have a different type?

Reply:
Ok, so let's wait for actual reports 👍